### PR TITLE
Fixes backtick in the documentation

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by turbaszek — APPROVED** [review]

File: docs/start.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ The installation is quick and straightforward.
     airflow webserver --port 8080
 
     # start the scheduler
-    # open a new terminal or else run webserver with `-D` option to run it as a deamon
+    # open a new terminal or else run webserver with ``-D`` option to run it as a deamon
```
Comment (kaxil) [inline docs/start.rst:49]: Isn't this just a comment, that won't be rendered?

image

**Comment by potiuk** [comment]
It was failing the static checks

**Comment by potiuk** [comment]
so we had failing master @kaxil :)

**Comment by kaxil** [comment]
aah, I didn't see this commit that added that line https://github.com/apache/airflow/commit/de5e8905e582a86c49bcfc6195d4731aa6be6c48

Thanks for fixing it -- these days, thanks to "Super Fast" Github Action Queues -- it is difficult to know the current status lol

**Comment by potiuk** [comment]
Indeed!

**Pull request merged**